src/pull_gtfs.py

```diff
@@ -147,8 +147,9 @@ def get_feed_metadata(self):
 
 class FeedManager:
     """Persistently manages information about feeds and whether we have their data."""
-    def __init__(self, db_filename):
+    def __init__(self, db_filename, workers=24):
```

Comment:
should we default to 24? Does multiprocessing handle available cores < workers gracefully?

Reply:
Changed default to CPU count.